capacity/encode_change_request.js

```diff
@@ -5,6 +5,7 @@ const channelIdRecordType = '2';
 const decreaseRecordType = '3';
 const increaseRecordType = '4';
 const requestIdRecordType = '1';
+const publicPrivateRecordType = '5';
```

Comment:
I think this should be `channel_flags` to match https://github.com/lightning/bolts/blob/master/02-peer-protocol.md#the-open_channel-message BOLT 02, and the subscribe to open requests API